frontend/wallet/src/render/components/SettingsPage.re

```diff
@@ -24,9 +88,25 @@ let make = () => {
          | Loading => React.string("...")
          | Error(err) => React.string(err##message)
          | Data(data) =>
-           <span className=Theme.Text.Body.regular>
-             {React.string("Wallet version: " ++ data##version)}
-           </span>
+           <>
+             <span className=Theme.Text.Body.regular>
+               {React.string("Wallet version: " ++ data##version)}
+             </span>
+             <div className=Styles.walletItemContainer>
+               {data##ownedWallets
+                |> Array.mapi(~f=(i, w) =>
+                     <>
```

Comment:
qq - is `<>` just short hand for div?

Reply:
That's actually ReasonReact for Fragments ([see here](https://reactjs.org/docs/fragments.html)).